src/pages/index.js

```diff
@@ -1,13 +1,42 @@
 import React from 'react'
 import Link from 'gatsby-link'
 
-const IndexPage = () => (
-  <div>
-    <h1>Hi people</h1>
-    <p>Welcome to your new Gatsby site.</p>
-    <p>Now go build something great.</p>
-    <Link to="/page-2/">Go to page 2</Link>
-  </div>
-)
+export default ({ data }) => {
+  console.log(data)
```

Comment:
We're not lumberjacks!

Reply:
don't crush my dreams...

![image](https://user-images.githubusercontent.com/5697474/30933231-9a02cd2e-a38f-11e7-9477-ed1eb76d99f8.png)